app/v1/controllers/requestParsers/validators/validations/SourceValidation.scala

```diff
@@ -25,7 +25,6 @@ object SourceValidation {
     def validate(source: Option[String]): List[MtdError] = {
       source match {
         case Some(x) if sources.contains(Some(x).get) => NoValidationErrors
-        case None => NoValidationErrors
```

Comment:
Wouldn't removing the None case make it so it gives an error even when there isn't a source? Since it should afterwards give a 'latest' source?

Reply:
The None case is impossible here as the swap is done in the Request Parser, and I've added a test for that there.